pyproject.toml

```diff
@@ -86,4 +86,4 @@ max-complexity = 10
 skip = "build,lib,venv,icon.svg,.tox,.git,.mypy_cache,.ruff_cache,.coverage"
 
 [tool.pyright]
-extraPaths = ["./charms/worker/k8s/lib"]
+extraPaths = ["./charms/worker/k8s/lib", "./charms/worker/k8s/src"]
```

Comment:
This is so that `from charm import K8sCharm` gets resolved by Pylance in the test file.